server/xpub-model/entities/manuscript/index.test.js

```diff
@@ -401,6 +402,72 @@ describe('Manuscript', () => {
     })
   })
 
+  describe('validate()', () => {
```

Comment:
I don't think we can add this test as it's for the new validate function which isn't included in this PR.